optimize/backend/src/it/java/io/camunda/optimize/service/importing/ZeebeUserTaskImportIT.java

```diff
@@ -546,9 +546,22 @@ public void importUnclaimOperation_viaWriter() {
         deployAndStartInstanceForProcess(
             createSimpleNativeUserTaskProcessWithAssignee(TEST_PROCESS, null, ASSIGNEE_ID));
     waitUntilUserTaskRecordWithElementIdExported(USER_TASK);
-    zeebeExtension.unassignUserTask(
-        getExpectedUserTaskInstanceIdFromRecords(getZeebeExportedUserTaskEvents()));
-    waitUntilUserTaskRecordWithIntentExported(ASSIGNED);
+
+    final boolean isZeebeVersionSnapshotForHigherThan870 =
```

Comment:
```suggestion
    final boolean isZeebeVersionHigherThan870 =
```
I think SNAPSHOT is already implicitly included in the "higher" part of this description